### DISCUSSION TITLE
19.03.4 Docker pull the v9.4.0 skywalking-oap-server docker image returns error

### DISCUSSION THREAD

**Discussion by zimu312500** [apache/skywalking#10838] [Q&A] [answered]
I  use this cmd to pull the image:docker pull apache/skywalking-oap-server:9.4.0 ，but return Error response from daemon: manifest for apache/skywalking-oap-server:v9.4.0 not found: manifest unknown: manifest unknown。My docker version as follows:
[root@master1 hik]# docker version
Client: Docker Engine - Community
 Version:           19.03.4
 API version:       1.40
 Go version:        go1.12.10
 Git commit:        9013bf583a
 Built:             Fri Oct 18 15:52:22 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.4
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.10
  Git commit:       9013bf583a
  Built:            Fri Oct 18 15:50:54 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.10
  GitCommit:        b34a5c8af56e510852c35414db4c1f4fa6172339
 runc:
  Version:          1.0.0-rc8+dev
  GitCommit:        3e425f80a8c931f88e6d94a8c831b9d5aa481657
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683

How to fix it ?

**Comment by Superskyyy** [comment, marked as answer]
@zimu312500 the image is there in dockerhub and the command works fine. It might be your dockerhub mirror's problem or network related.

**Reply by zimu312500** [reply]
docker pull apache/skywalking-oap-server:9.2.0，when I use this command to pull the 9.2.0 image,it's right, but pull the 9.4.0 image occured upper errors，I don't think it's my dockerhub mirror's problem.

**Reply by lujiajing1126** [reply]
> docker pull apache/skywalking-oap-server:9.2.0，when I use this command to pull the 9.2.0 image,it's right, but pull the 9.4.0 image occured upper errors，I don't think it's my dockerhub mirror's problem.

According to the log `Error response from daemon: manifest for apache/skywalking-oap-server:v9.4.0 not found: manifest unknown: manifest unknown`, you were using `v9.4.0` instead of `9.4.0`. Pls carefully recheck the tag.
